src/components/tweet.js

```diff
@@ -62,6 +64,92 @@ export default class Tweet extends React.Component {
     });
   }
 
+  openMediaInWindow (med) {
+    let image = new Image();
+    let loaded = false;
+    var mediaUrl = 0;
+    var media = med.video_info;
+    if (med.video_info != null) {
+      mediaUrl = med.video_info.variants[0].url;
+      var mediaType = med.video_info.variants[0].content_type;
+    } else {
+      mediaUrl = med.media_url;
+    }
+    image = document.getElementById('tweet_mediaid');
+    image.onload = function () { loaded = true; };
+    let win = new BrowserWindow({titleBarStyle: 'hidden', x:(screen.width/2)-(450/2),y:(screen.height/2)-(450/2), resizable: false, width: 100, height: 100});
+    win.loadURL('file://' + __dirname + '../..' + '/mediaPopup.html');
+    if (loaded) {
+      clearInterval(wait);
+    } else {
+      win.webContents.executeJavaScript(`
```

Comment:
Can't you avoid using `executeJavaScript`? Nocturn has some code around ipc and browser window.